tools/st2_deploy.sh

```diff
@@ -4,11 +4,15 @@ function version_ge() { test "$(echo "$@" | tr " " "\n" | sort -V | tail -n 1)"
 
 if [ -z $1 ]
 then
-  VER='0.6.0'
```

Comment:
I always thought this was replaced somewhere in the build pipeline, but I'm probably wrong and someone just manually updated scripts on ops.stackstorm.net :P